src/shipit_pulse_listener/requirements_frozen.txt

```diff
@@ -26,3 +26,4 @@ structlog==17.2.0
 taskcluster==1.3.3
 urllib3==1.21.1
 yarl==0.10.2
+flake8-coding
```

Comment:
this should be ``flake8-coding==1.3.0`` and also sorted (should come after ``flake8==...``